src/import_edf.jl

```diff
@@ -208,81 +302,264 @@ function groupby(f, list)
     return d
 end
 
-"""
-    extract_channels_by_label(edf::EDF.File, signal_names, channel_names;
-                              unit_alternatives=STANDARD_UNITS, preprocess_labels=(l,t) -> l)
+# unpack a single channel spec from labels:
+# "channel"
+canonical_channel_name(channel_name) = channel_name
+# "channel" => ["alt1", "alt2", ...]
+canonical_channel_name(channel_alternates::Pair) = first(channel_alternates)
 
-For one or more signal names and one or more channel names,
-return a list of `[(infos, errors)...]` where `errors` is a vector of errors that occurred,
-and `infos` is a vector of `(si::Onda.SamplesInfo, edf_signals::Vector{EDF.Signal})`,
-where `edf_signals` align with `si.channel`. This list can have more than one pair
-if channels with the same signal kind/type have different sample rates or
-physical units.
+plan_edf_to_onda_samples(signal::EDF.Signal, s; kwargs...) = plan_edf_to_onda_samples(_named_tuple(signal.header), s; kwargs...)
```

Comment:
Doesn't actually matter (feel free to ignore) but I'd have this method call the other, since you're saying that the way to deal with a signal is to treat it like its header.
```suggestion
plan_edf_to_onda_samples(signal::EDF.Signal, s; kwargs...) = plan_edf_to_onda_samples(signal.header, s; kwargs...)
```